# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/slmodem/kernelver.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN slmodem-2.9.10/Makefile slmodem-2.9.10-new/Makefile
--- slmodem-2.9.10/Makefile	2004-08-08 15:27:51.000000000 +0200
+++ slmodem-2.9.10-new/Makefile	2005-01-04 12:36:01.202191928 +0100
@@ -34,7 +34,7 @@
 	$(RM) -rf ${DESTDIR}/var/lib/slmodem
 
 drivers:
-	$(MAKE) -C drivers KERNEL_DIR=$(KERNEL_DIR)
+	$(MAKE) -C drivers KERNEL_VER=$(KERNEL_VER)
 
 install-drivers: drivers
 	$(MAKE) install -C drivers KERNEL_DIR=$(KERNEL_DIR)
